starter_code/queries.md

```diff
@@ -4,76 +4,76 @@
 
 ### 1. All the companies whose name match 'Babelgum'. Retrieve only their `name` field.
 
-<!-- Your Code Goes Here -->
+db.companies.find({name:"Babelgum" },{name:1, _id: 0} )
 
 ### 2. All the companies that have more than 5000 employees. Limit the search to 20 companies and sort them by **number of employees**.
 
-<!-- Your Code Goes Here -->
+db.companies.find(  {number_of_employees: {$gt: 5000}}  ).sort({name:1})
 
```

Comment:
"Limit the search to 20 companies and sort them by number of employees." (in ascending or descending order, it's not specified):
=> `.limit(20).sort({number_of_employees: -1})`